app/src/debug/res/raw/initial_experiments.json

```diff
@@ -4,46 +4,53 @@
           "appId": "org.mozilla.fenix",
           "appName": "fenix",
           "channel": "nightly",
-          "branches": [{
-                  "slug": "control",
-                  "ratio": 100,
-                  "feature": {
-                      "value": {},
-                      "enabled": true,
-                      "featureId": "nimbus-validation"
-                  }
-              },
+          "branches": [
               {
-                  "slug": "fancy-settings",
+                  "slug": "no-mr2",
                   "ratio": 0,
                   "feature": {
                       "value": {
-                          "settings-title": "Fancy Settings"
+                           "sections-enabled": {
+                               "recentExplorations": true,
+                               "recentlySaved": false,
+                               "jumpBackIn": false,
+                               "pocket": false
+                          }
```

Comment:
Is this the schema you have settled on? If so, I'm happy to add this to Experimenter so we can test from there, also.

Reply:
I'm not sure, but I think the names reflect the features that we want to tests.

Reply:
I guess it's OK to use these names

Reply:
It's updated now :) 
https://github.com/mozilla-mobile/fenix/pull/21411/files#diff-cfdb9388f7e08573f26394a641d2e28d18b862b11553fd5b3e7c30edd1837ccfR14